components/LocalMoran.jsx

```diff
@@ -0,0 +1,136 @@
+import React, {useEffect, useCallback, useRef} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import {
+  addDataToMap,
+  layerVisualChannelConfigChange,
+  layerVisConfigChange,
+  forwardTo
+} from 'kepler.gl/actions';
+import KeplerGl from 'kepler.gl';
+import jsgeoda from 'jsgeoda';
+import {processGeojson} from 'kepler.gl/dist/processors';
+import {maybeToDate} from 'kepler.gl/dist/utils';
+import {rgbToHex} from 'kepler.gl/dist/utils/color-utils';
+
+const MAPBOX_TOKEN = process.env.NEXT_PUBLIC_MAPBOX_TOKEN;
```

Comment:
Suggestion: the MAPBOX_TOKEN has been used in more than one file. You can put it in a file where all constants are defined, and import it from the constant file.